rasa/core/policies/rule_policy.py

```diff
@@ -123,83 +277,25 @@ def _rule_is_good(
             )
         )
 
-    def train(
-        self,
-        training_trackers: List[DialogueStateTracker],
-        domain: Domain,
-        interpreter: NaturalLanguageInterpreter,
-        **kwargs: Any,
-    ) -> None:
-        """Trains the policy on given training trackers."""
-        self.lookup = {}
-
-        # only use trackers from rule-based training data
-        training_trackers = [t for t in training_trackers if t.is_rule_tracker]
-
-        # only consider original trackers (no augmented ones)
-        training_trackers = [
-            t
-            for t in training_trackers
-            if not hasattr(t, "is_augmented") or not t.is_augmented
-        ]
-        (
-            trackers_as_states,
-            trackers_as_actions,
-        ) = self.featurizer.training_states_and_actions(training_trackers, domain)
-
-        self._add_states_to_lookup(trackers_as_states, trackers_as_actions, domain)
-
-        # remove action_listens that were added after conditions
-        updated_lookup = self.lookup.copy()
-        for key in self.lookup.keys():
-            # Delete rules if there is no prior action or if it would predict
-            # the `...` action
-            if "prev" not in key or self.lookup[key] == domain.index_for_action(
-                RULE_SNIPPET_ACTION_NAME
-            ):
-                del updated_lookup[key]
-            elif RULE_SNIPPET_ACTION_NAME in key:
-                # If the previous action is `...` -> remove any specific state
-                # requirements for that state (anything can match this state)
-                new_key = re.sub(r".*prev_\.\.\.[^|]*", "", key)
-
-                if new_key:
-                    if new_key.startswith("|"):
-                        new_key = new_key[1:]
-                    if new_key.endswith("|"):
-                        new_key = new_key[:-1]
-                    updated_lookup[new_key] = self.lookup[key]
-
-                del updated_lookup[key]
-
-        self.lookup = updated_lookup
-        logger.debug("Memorized {} unique examples.".format(len(self.lookup)))
-
-    def predict_action_probabilities(
-        self,
-        tracker: DialogueStateTracker,
-        domain: Domain,
-        interpreter: NaturalLanguageInterpreter = RegexInterpreter(),
-        **kwargs: Any,
-    ) -> List[float]:
-        """Predicts the next action the bot should take after seeing the tracker.
+    @staticmethod
+    def _predict_default_action(tracker: DialogueStateTracker) -> Optional[Text]:
```

Comment:
optional: the name of the function sounds like it returns a bool
```suggestion
    def _applicable_default_action(tracker: DialogueStateTracker) -> Optional[Text]:
```